api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -33,4 +33,18 @@
     	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments" columnDataType="varchar(1024)"/>
     </changeSet>
 
+    <changeSet id="TRUNK-5413" author="odorajonathan" dbms="mysql">
+      <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="order_set" columnName="category"/></not>
+      </preConditions>
+      <comment>
+          Add Category field to OrderSet
+      </comment>
+      <addColumn tableName="order_set">
+            <column name="category" type="int"/>
+      </addColumn>
+      <addForeignKeyConstraint constraintName="category_fk" referencedTableName="concept" 
```

Comment:
For the constraint name, can you add the table name to avoid name conflicts for the same field being used else where in another table? Something like category_order_set_fk